README.md

```diff
@@ -1,58 +1,55 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
-
-Although you can run Micromasters locally with a default sqlite database after
-installing the ``requirements.txt`` file, the recommended way is to 
-use [Docker](https://www.docker.io). Install Docker, then install 
-``docker-compose`` and run the app: 
+## Major Dependencies
+- pip, virtualenv, homebrew (OSX only)
+- Postgres >= 9.4
+  - OSX recommended install method: [Postgres.app](http://postgresapp.com/)
+- Python 3
+  - OSX recommended install method: Homebrew `brew install python3`
+- Docker (_optional, but strongly recommended_)
+  - OSX recommended install method: [Download from Docker website](https://docs.docker.com/mac/)
+- docker-compose (_if you're using Docker, this is required_)
+  - Recommended install: pip `pip install docker-compose`
 
-    pip install docker-compose 
-    docker-compose up
+## Getting Started
 
-### (OSX only) Create your docker machine:
+1. Create and activate a virtualenv in the root folder of the Micromasters project directory.
+1. Install the python dependencies ``pip install -r requirements.txt``
```

Comment:
could we divide this into Linux and Mac OS sections? The Linux section will only really say 'install docker, docker-compose and do `docker-compose up`' but it would good to be totally clear that these instructions are specific to OSX.